app/forms/assessor_interface/verify_professional_standing_form.rb

```diff
@@ -1,9 +0,0 @@
-# frozen_string_literal: true
-
-class AssessorInterface::VerifyProfessionalStandingForm
```

Comment:
I'm not sure we should be removing this form as it's still used:

https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/2e9a6e6274bf20a845db934a9decc8a680d4a0e0/app/controllers/assessor_interface/assessment_recommendation_verify_controller.rb#L38-L54